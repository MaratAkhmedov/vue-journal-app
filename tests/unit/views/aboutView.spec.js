import AboutView from '@/views/AboutView.vue';
import { shallowMount } from '@vue/test-utils'

describe('AboutView test', () => {
  it('should match with a snapshot', () => {
    const wrapper = shallowMount(AboutView);
    expect(wrapper.html()).toMatchSnapshot()
  });
})