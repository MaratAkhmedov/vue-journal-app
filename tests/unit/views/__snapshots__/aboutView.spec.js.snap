// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView test should match with a snapshot 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
